internal/lookout/ui/src/containers/JobsContainer.tsx

```diff
@@ -34,6 +34,7 @@ export type JobsContainerState = {
   jobDialogIsOpen: boolean
   clickedJob?: Job
   getJobsRequestStatus: RequestStatus
+  abortController: AbortController
```

Comment:
Why is there no import of this type/class anywhere in the code?  

I'd expect to see an import somewhere of AbortController?

Reply:
Its apparently part of react?